src/Form/Form.php

```diff
@@ -281,7 +281,7 @@ public function execute(array $data, array $options = []): bool
      * @param array $data Form data.
      * @return bool
      */
-    protected function _execute(array $data): bool
+    protected function shouldExecute(array $data): bool
```

Comment:
similarly here

Reply:
Should sounds like it is just checking not doing.

Reply:
right, i misunderstood that method as a hook to tell, if the perform should be executed or not... will rename that to `doExecute`

Reply:
Or run() or alike